### PR TITLE
Filter kills only fight (fixes #92)

### LINKED ISSUE #92: Being able to filter on "Kills only" when selecting an encounter

### PR DESCRIPTION
**Pull request by Yuyz0112**
- [x] Use `Toggle` to filter kills in `FightSelector`, also changed a class name of toggle wrapper.
- [x] Use a `renderContent` method instead of create an IIFE every render.

### CODE REVIEW

File: src/Main/App.js [hunk 1]
```diff
@@ -85,6 +85,7 @@ class App extends Component {
 
     this.handleReportSelecterSubmit = this.handleReportSelecterSubmit.bind(this);
     this.handleRefresh = this.handleRefresh.bind(this);
+    this.renderContent = this.renderContent.bind(this);
```
Comment (MartijnHols) [inline src/Main/App.js]: This is only necessary for event handlers that you bind to elements, it can be removed.

File: src/Main/App.js [hunk 2]
```diff
@@ -285,6 +286,49 @@ class App extends Component {
       .then(response => response.json());
   }
 
+  renderContent() {
```
Comment (MartijnHols) [inline src/Main/App.js]: Please put this method above the `render()` method so that everything can be found easily.

**Comment by Yuyz0112** [comment]
done

**Comment by MartijnHols** [comment]
Thanks!

**Pull request merged**